### PR TITLE
feat: standardise focus visible a11y on Isomer components

### PR DESCRIPTION
**Pull request by karrui**
Diff too large, Graphite cannot generate sad.

* increase touchpoint of siderail expand button
* add focus-visible highlight to breadcrumb
* add focus-visible highlight to button

focus-visible highlight is the bright ass yellow. Added a new token `colors.utility.highlight` for standardisation.

focus-visible means only seen when user uses keyboard navigation

### CODE REVIEW

**Comment by karrui** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/opengovsg/isomer/548?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#548** <a href="https://app.graphite.dev/github/pr/opengovsg/isomer/548?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#529** <a href="https://app.graphite.dev/github/pr/opengovsg/isomer/529?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @karrui and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by seaerchin** [comment]
got issue, ss-ed here for your convenience
image

**Comment by karrui** [comment]
> got issue, ss-ed here for your convenience
> image
> 

Grrr yup not an issue it’s because I imported rac stuff. Easily fixed! Review as if no errors please!

**Review by seaerchin — APPROVED** [review]
seems like most changes are styling for focus visible, with some extra changes for sceen readers!

most changes seem to be just copypasting old styles and runnign through `rac` stuff, for these i just compared and checked that old styling was mostly preserved.

lgtm, just going to check against chromatic for the few i've noted down!

File: packages/components/src/templates/next/components/complex/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -1,16 +1,22 @@
+"use client"
```
[the remark this answers is not in the document]
Comment (seaerchin) [reply packages/components/src/templates/next/components/complex/Accordion/Accordion.tsx:1]: searched against `react-spectrun`. i found [this](https://github.com/adobe/react-spectrum/discussions/5767) answer!

File: packages/components/src/templates/next/components/internal/BaseParagraph/BaseParagraph.tsx [hunk 2]
```diff
@@ -12,7 +14,7 @@ export const BaseParagraph = ({
   return (
     <p
       className={twMerge(
-        `hover:[&_a]text-link-hover [&:not(:first-child)]:mt-6 [&:not(:last-child)]:mb-6 after:[&_a[target*="blank"]]:content-['_↗'] [&_a]:text-link [&_a]:underline`,
+        `hover:[&_a]text-link-hover [&:not(:first-child)]:mt-6 [&:not(:last-child)]:mb-6 after:[&_a[target*="blank"]]:content-['_↗'] [&_a]:text-link [&_a]:underline [&_a]:outline-none focus-visible:[&_a]:bg-utility-highlight focus-visible:[&_a]:text-base-content-strong focus-visible:[&_a]:decoration-transparent focus-visible:[&_a]:shadow-focus-visible focus-visible:[&_a]:transition-none`,
```
[the remark this answers is not in the document]
Comment (karrui) [reply packages/components/src/templates/next/components/internal/BaseParagraph/BaseParagraph.tsx:17]: changes text color, bg color, and adds the yellow and black underline (as a box shadow)

File: packages/components/src/templates/next/components/internal/Footer/Footer.tsx [hunk 3]
```diff
@@ -33,31 +39,52 @@ const SiteNameSection = ({ siteName }: Pick<FooterProps, "siteName">) => {
...
   url,
 }: FooterItemType & Pick<FooterProps, "LinkComponent">) => {
   if (url.startsWith("http")) {
     return (
-      <LinkComponent
+      <BaseLink
+        LinkComponent={LinkComponent}
         href={url}
         target="_blank"
         rel="noopener noreferrer nofollow"
```
[the remark this answers is not in the document]
Comment (karrui) [reply packages/components/src/templates/next/components/internal/Footer/Footer.tsx:69]: ya already tracked in ISOM-1430

**Comment by karrui** [comment]
### Merge activity

* **Sep 5, 4:43 AM EDT**: @karrui started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/opengovsg/isomer/548).

**Pull request merged**